vignettes/rebuild_cached_data.Rmd

```diff
@@ -118,7 +118,8 @@ updated_files <- tryCatch(
   system(git_call, intern = TRUE),
   error = function(e) e
 )
-if (is(e, "error")) {
+status_uf <- attr(updated_files, "status")
+if (is(updated_files, "error") || (!is.null(status_uf) && status_uf == 1)) {
```

Comment:
I tested with git not working and it sometimes will produce a warning: 
```
Warning in system(git_call, intern = TRUE) :
  running command 'git diff origin/main --name-only' had status 1
```
instead of an error, so just wanted to cover that as well.

But other than that everything seems to be working, so if you think it's fine we can merge now!